doc/manual/src/release-notes/rl-next.md

```diff
@@ -10,3 +10,43 @@
   This avoids a lot of spurious errors where some benign strings end-up having
   a context just because they are read from a store path
   ([#7260](https://github.com/NixOS/nix/pull/7260)).
+
+* Nix can now automatically pick UIDs for builds, removing the need to
+  create `nixbld*` user accounts. these UIDs are allocated starting at
+  872415232 on Linux and 56930 on macOS.
```

Comment:
Would it be possible to add / change these to the hex values? Somebody grepping the codebase for this magic number won't find it since you changed it to hex.